src/utils/connectors/postgres.py

```
@@ -226,21 +296,64 @@ def decorator(fn):
+            for attempt_number in range(1, maximum_attempts + 1):
+                if delay is not None:
+                    time.sleep(delay)
+                    if reconnect_pool:
+                        try:
+                            self.connect()
+                        except osmo_errors.OSMOConnectionError as error:
+                            if attempt_number == maximum_attempts:
+                                _log_postgres_retry_exhausted(
+                                    fn.__name__, attempt_number, maximum_attempts, error)
+                                raise osmo_errors.OSMODatabaseError(
+                                    f'Error: {str(error)}') from error
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)